doc/servermd/AnalyticsServerGuide_en_US.md

```diff
@@ -1,6 +1,6 @@
 Using Open WebRTC Toolkit For Media Analytics
 =============================================
-In this document we introduce the media analytics functionality provided by Open WebRTC Toolkit (https://github.com/open-webrtc-toolkit/owt-server), namely OWT, and a step by step guide to implement your own media analytics plugins with GStreamer pipeline and Intel Distribution of OpenVINO.
+In this document we introduce the media analytics functionality provided by Open WebRTC Toolkit (https://github.com/open-webrtc-toolkit/owt-server), namely OWT, and a step by step guide to implement your own media analytics pipeline with GStreamer pipeline and Intel Distribution of OpenVINO.
 
```

Comment:
implement your own media analytics pipeline with GStreamer and Intel Distribution of OpenVINO.